features/ua-client-hints.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2024-10-21
-name: User-Agent Client Hints
-description: TODO
+name: User agent client hints
+description: User agent (UA) client hint headers allow a server to vary responses based on the user agent (browser), operating system, and device.
```

It's not just the server. The client can use the hints too with `navigator.userAgentData` API. How about something like this:

```suggestion
description: The `Sec-CH-UA` HTTP request header and the `navigator.userAgentData` API expose browser and platform information, which the web server and client can use to vary their responses and code logic. User agent client hints are similar but more privacy-preserving and more robust than user agent strings.
```

---

Got it and tweaked it a bit to avoid talking about server and client 

```suggestion
description: The `Sec-CH-UA` HTTP request header and the `navigator.userAgentData` API expose browser and platform information used to vary responses and application logic. User agent client hints are similar to user agent strings but more privacy-preserving and more robust.
```